content/blog/2018-03-22-five-practical-tips-for-shipping-software-faster.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 5 Practical Steps to Shipping Better Software, Faster
+sub_title: All software development teams are unique. They are made up of developers, project managers, quality assurance testers, and designers of vary skill sets and seniority levels. 
+tags: [software, speed]
+date: 2018-03-22
+publishdate: 2018-03-22
+hero_image: /images/blog/2018-03-22-five-practical-tips-for-shipping-software-faster/hero.png
+author:
+  name: Kevin Coleman
+  url: https://twitter.com/KevFColeman
+  mail: kevin@meshstudio.io
+  avatar: "https://pbs.twimg.com/profile_images/911288822217924608/Oq631azq_400x400.jpg"
+---
+
+# 5 Practical Steps to Shipping Better Software, Faster
+
+All software development teams are unique. They are made up of developers, project managers, quality assurance testers, and designers of vary skill sets and seniority levels. 
+
+Despite the differences in their makeup, all of these teams share a common objective: *to ship high software quality products, fast*. 
+
+In reality, the overwhelming majority of software development teams are not able to perform at this high of a level. Under performance in development teams  typically is not due to a lack of skill nor a lack of desire, but rather, due to flaws in their software development process. 
```

Comment:
Under-performance in development teams isn't typically due to a lack of skill or desire, but rather flaws in the team's software development process*